modules/retail/pages/retail-install-setup.adoc

```diff
@@ -149,23 +154,32 @@ salt <branch_server_salt_id> state.apply image-sync
 . The image details will be transferred to [path]``/srv/saltboot`` on the branch server.
 
 
+[[retail-install-setup-11sp4terminals]]
+== Terminals based on {slsa}{nbsp}11 SP4
 
-== Terminals based on {slsa}{nbsp}11 SP{nbsp}3
 
-POS Terminals based on {sls}{nbsp}11 SP{nbsp}3 can be deployed in much the same way as other terminals, with a few differences.
+POS Terminals based on {sls}{nbsp}11 SP{nbsp}4 can be deployed in much the same way as other terminals, with a few differences.
 
 * You must use the {slsa}{nbsp}11 template
-* {slsa}{nbsp}11 images need to be activated with the [systemitem]``SLES11 SP3 i586`` and [systemitem]``SLEPOS 11 SP3 i586`` channels
+* {slsa}{nbsp}11 images need to be activated with the [systemitem]``SLES11 SP4 i586`` and [systemitem]``SLEPOS 11 SP4 i586`` channels
 
```

Comment:
It should really be SLES11 SP3 i586 because of LTSS, see
https://github.com/SUSE/doc-susemanager/pull/472#discussion_r288090193

Reply:
I do not completely understand it.  Would it be enough if I'd just revert line 164?  Or all the other SLE 11 SP3/SP4 related changes in this file as well? @nadvornik

Reply:
revert whole section -  from line 157 to 185

Reply:
ok, done